mod/au_subgroups/views/default/au_subgroups/search_results.php

```diff
@@ -24,7 +24,7 @@
 }
 else {
   $query = sanitize_string($vars['q']);
-  $options['wheres'][] = "g.name LIKE '{$query}%'";
+  $options['wheres'][] = "g.name LIKE '%{$query}%'";
```

Comment:
That's a much slower search, we might have to maybe have it do
```php
 g.name LIKE '{\"en\":\"{$query}%'";
```
if this is too slow on prod